ffi/dart/test/icu_test.dart

```diff
@@ -61,6 +61,25 @@ void main() {
 
     var locale = Locale.fromString('de');
 
+    var builder = DateTimeFieldSetBuilder(
+      dateFields: DateFields.mde,
+      length: DateTimeLength.short,
+      timePrecision: TimePrecision.minute,
+      zoneStyle: null,
+      alignment: null,
+      yearStyle: null,
```

Comment:
Note: The generated code seems to make it required that I set these fields, even though their type is an `Option`.